internal/configs/virtualserver.go

```diff
@@ -359,6 +359,7 @@ func (vsc *virtualServerConfigurator) GenerateVirtualServerConfig(
 	var vsrErrorPagesRouteIndex = make(map[string]int)
 	var vsrLocationSnippetsFromVs = make(map[string]string)
 	var vsrPoliciesFromVs = make(map[string][]conf_v1.PolicyReference)
+	IsVSR := false
```

Comment:
There doesn't seem to be a need to export `IsVSR` in this context?
```suggestion
	isVSR := false
```